pkg/nsx/services/vpc/builder.go

```diff
@@ -32,15 +32,15 @@ func generateIPBlockSearchKey(cidr string, vpcCRUID string) string {
 	return cidr + "_" + vpcCRUID
 }
 
-func buildPrivateIpBlock(vpc *v1alpha1.VPC, cidr, ip, project, cluster string) model.IpAddressBlock {
-	suffix := vpc.GetNamespace() + "-" + vpc.Name + "-" + ip
+func buildPrivateIpBlock(networkInfo *v1alpha1.NetworkInfo, cidr, ip, project, cluster string) model.IpAddressBlock {
+	suffix := networkInfo.GetNamespace() + "-" + networkInfo.Name + "-" + ip
```

Comment:
we may not use networkInfo.Name here, since now one network info contains multiple vpc, it is possible that the name duplicate with another vpc under the same network info